quint-assign1/quint-assign1.py

```diff
@@ -0,0 +1,218 @@
+"""
+Assignment 1: Cryptography
+Course: CS 92SI
+Name: Quint Underwood
+Date: 9 October 2015
+
+Replace this with a description of the program.
+"""
+
+def encrypt_caesar(plaintext):
+    """
+    Encrypts plaintext using a Caesar cipher.
+    Add more implementation details here.
+    """
+    shift = int(input("By how many letters do you want to shift? "))
+    encoded_phrase = ""
+    for letter in plaintext:
+        letter = chr(ord(letter) + shift)
+        if letter > 'z':
```

Comment:
You run into a problem here with uppercase characters because `'Z' + 3` will still be less than `'z'`